crds/configmanagement_v1_configmanagement.yaml

```diff
@@ -63,6 +63,10 @@ spec:
                   all ConfigSync pods. This field is intentionally left hidden/undocumented
                   since it is really only used to gather extra logs for support cases.
                 type: integer
+              allowVerticalScale:
```

Comment:
is this not merged into main yet? https://github.com/GoogleContainerTools/kpt-config-sync/search?q=allowVerticalScale

Reply:
This is for 1.12.2 release, in 1.12.2 we add a new field to Config Managment CRD, so  I need to update it in anthos-config-management-samples folder

Reply:
This field is in nomos-operator, it is a separate repo.

Reply:
The release process of the nomos-operator creates patch releases from `HEAD`, instead of from a release branch and then cherrypicks. This would leak new features like this into patch releases. We should improve the nomos-operator release process.

Reply:
Sounds good.